eclcli/common/utils.py

```diff
@@ -123,6 +124,8 @@ def get_item_properties(item, fields, mixed_case_fields=None, formatters=None):
         else:
             field_name = field.lower().replace(' ', '_')
         data = getattr(item, field_name, '')
+        if data == '' and field_name == 'project_id':
+            data = getattr(item, 'tenant_id', '')
```

Comment:
Fixed that `project_id` was empty in the output result of network type resources